src/content/pages/index.md

```diff
@@ -2,32 +2,148 @@
+    earn rewards.
+  image: /images/staking-threshold.png
+  buttons:
+    - label: Start Staking
+      url: https://dashboard.threshold.network/overview/network
+      variant: EXTERNAL_SOLID
+    - variant: INTERNAL_OUTLINE
+      label: Learn More
+      url: /earn/staker
+  rowReverse: false
+  bgColor: "#858585"
+lpRole:
+  title: Provide Liquidity
+  description: Liquidity providers earn yields by depositing assets into liquidity pools.
+  image: /images/liquidity.png
+  buttons:
+    - label: Learn More
+      url: /earn/liquidity-provider
+      variant: INTERNAL_SOLID
+  rowReverse: true
+  bgColor: "#858585"
+btcRole:
+  title: Bridge BTC to Ethereum
+  description: tBTC is Threshold’s decentralized bridge to bring BTC to the
+    Ethereum network.
+  image: /images/tbtc.png
+  buttons:
+    - label: Mint tBTC
+      url: NEED_URL
+      variant: EXTERNAL_SOLID
+    - label: Learn More
+      url: /earn/btc
+      variant: INTERNAL_OUTLINE
+  rowReverse: false
+  bgColor: "#333333"
+tokenHolderRole:
+  title: Token Holder DAO
+  description: Make the most of your T tokens on the Threshold Network by
+    participating in DAO governance.
+  image: /images/token-holder.png
+  buttons:
+    - label: Governance Forum
+      url: NEED_URL
+      variant: EXTERNAL_SOLID
+    - label: Learn More
+      url: /earn/token-holder
+      variant: INTERNAL_OUTLINE
+  rowReverse: true
+  bgColor: "#3c3c3c"
+migrationInfo:
+  title: Do you own KEEP or NU?
+  description: Keep and NuCypher merged to form the Threshold Network. Migrate
+    your tokens to T!
+  image: /images/legacy-tokens.png
+  buttons:
+    - label: Migrate tokens
+      url: https://dashboard.threshold.network/
+      variant: EXTERNAL_OUTLINE
+  bgColor: "#141414"
+  rowReverse: false
+harnessThePower:
+  title: Harness the power of Threshold
+  description: Threshold leverages threshold cryptography to protect digital
+    assets by distributing operations across independent parties, requiring some
+    threshold number of them (t-of-n) to cooperate.
+  buttons:
+    - label: About Threshold
+      url: /about
+      variant: INTERNAL_SOLID
+  subitems:
     - title: Decentralized
-      icon:
-        image: /images/logo-buffer.svg
-        alt: Decentralized
-      body: Threshold is the only bridge protocol that is truly decentralized.
+      description: Threshold utilizes a network of independent nodes to provide
+        threshold cryptographic services without a central authority.
+      image: /images/decentralized-icon.png
     - title: Secure
-      icon:
-        image: /images/secure-icon.svg
-        alt: Secure
-      body: Threshold is audited by the strongest firms in the space.
+      description:
+        Splitting cryptographic operations across nodes increases security
+        and availability and reduces trust assumptions. Threshold is
+        [audited](/audits) by the best firms in the space.
```

Comment:
If it's a markdown we should inject the html via `dangerouslySetInnerHTML={{ __html: description }}` in component. Currently it renders as:

![image](https://user-images.githubusercontent.com/57687279/160820855-1c33c9c0-4eb5-47a8-a70f-050b68ed91f3.png)

Reply:
Done please see https://github.com/threshold-network/splash-page/pull/22.